pkg/notify/notifier/email/email.go

```diff
@@ -321,8 +322,7 @@ func (n *Notifier) getEmailConfig(e *nmconfig.Email) (*config.EmailConfig, error
 		}
 
 		ec.TLSConfig = tlsConfig
-		requireTLS := true
-		ec.RequireTLS = &requireTLS
+		ec.RequireTLS = &e.EmailConfig.RequireTLS
```

Comment:
I think `ec.RequireTLS = &e.EmailConfig.RequireTLS` can be deleted, because it had assigned before.
https://github.com/kubesphere/notification-manager/blob/master/pkg/notify/notifier/email/email.go#L263

Reply:
@wanjunlei I fixed it.